src/components/home/ActivityPhotos.module.scss

```
@@ -57,33 +60,36 @@
 
 // section 자체엔 폭 제한이 없어(Figma 원본 그대로), 카드가 많아지면 배경 바깥으로
 // 넘쳤다. marquee가 그 폭만큼만 보여주고 넘치는 부분은 잘라낸다.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)